test-cypress/integration/explore/visual.graph.spec.js

```diff
@@ -447,6 +447,33 @@ describe('Visual graph screen validation', () => {
         });
     });
 
+    it('Test can create custom visual graph', () => {
+        cy.visit('/graphs-visualizations');
+        getCreateCustomGraphLink().click();
+        cy.url().should('include', '/config/save');
+        getGraphConfigName().type('configName');
+
+        cy.get('.page-1-link').should('be.visible')
+            .and('contain', 'Starting point')
+            .and('have.class', 'active');
+        cy.get('.page-2-link').should('be.visible')
+            .and('contain', 'Graph expansion');
+        cy.get('.page-3-link').should('be.visible')
+            .and('contain', 'Node basics')
+        cy.get('.page-4-link').should('be.visible')
+            .and('contain', 'Edge basics');
+        cy.get('.page-5-link').should('be.visible')
+            .and('contain', 'Node extra');
+        cy.get('.page-2-link').click();
+        cy.get('.page-2-link').should('have.class', 'active');
+
+        cy.get('.expand-samples a').first().click();
```

Comment:
This selector could be `.expand-samples .list-group-item`

Reply:
Changed it.